blog/mkdocs.yml

```
@@ -0,0 +1,93 @@
+INHERIT: config/nav.yml
+
+site_name: Knative
+site_url: https://knative.dev/blog
+site_description: Knative Blog
+extra_css:
+  - stylesheets/extra.css
+
+repo_url: https://github.com/knative/docs
+edit_uri: edit/main/blog/docs
+
+
+theme:
+  name: material
+  docs_dir: blog/
+  logo: images/logo/rgb/knative-logo-rgb.png
+  favicon: images/favicon.ico
+  custom_dir: overrides
+  features:
+    - navigation.tabs
+    - navigation.tracking
+    - navigation.tabs.sticky
+    - navigation.indexes
+    - navigation.top
+
+markdown_extensions:
+  # - mdx_include:
+  #     base_path: docs
+  - pymdownx.emoji:
+      emoji_index: !!python/name:materialx.emoji.twemoji
+      emoji_generator: !!python/name:materialx.emoji.to_svg
+  - attr_list
+  - meta
+  - pymdownx.superfences
+  - pymdownx.tabbed
+  - pymdownx.details
+  - pymdownx.snippets:
+      base_path: docs/snippets
+  - pymdownx.keys
+  - pymdownx.caret
+  - pymdownx.mark
+  - pymdownx.tilde
+  - pymdownx.escapeall
+  - pymdownx.tasklist:
+      custom_checkbox: true
+  - admonition
+  - toc:
+      permalink: true
```

```suggestion
      permalink: true
      toc_depth: 1
```
can we do just the top level, this will benefit the release blog posts

---

done